components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -229,6 +229,7 @@
 <config_cvmix_kpp_use_enhanced_diff>.true.</config_cvmix_kpp_use_enhanced_diff>
 <config_cvmix_kpp_nonlocal_with_implicit_mix>.false.</config_cvmix_kpp_nonlocal_with_implicit_mix>
 <config_cvmix_kpp_use_theory_wave>.false.</config_cvmix_kpp_use_theory_wave>
+<config_cvmix_kpp_use_active_wave>.false.</config_cvmix_kpp_use_active_wave>
```

Comment:
is this why we point to a separate fork as you don't have this merged to mpas yet?

Reply:
@vanroekel, exactly. It's just a temporary until I complete the Stokes drift implementation and get it merged.

Reply:
got it that's what I thought but thought I'd confirm.